docs/user/get_started.md

```diff
@@ -0,0 +1,253 @@
+# Quick Start with garage
+
+Table of Content
+- [What is garage?](#what-is-garage)
+- [Why garage?](#why-garage)
+- [Kick Start garage](#kick-start-garage)
+- [Open Source Support](#open-source-support)
+- [Resources](#resources)
+
+
+## What is garage?
+
+garage is a reinforcement learning (RL) toolkit for developing and evaluating algorithms. The garage library also provides a collection of state-of-the-art implementations of RL algorithms.
+
+The toolkit provides a wide range of modular tools for implementing RL algorithms, including:
+
+- Composable neural network models
+- Replay buffers
+- High-performance samplers
+- An expressive experiment definition interface
+- Tools for reproducibility (e.g. set a global random seed which all components respect)
+- Logging to many outputs, including TensorBoard
+- Reliable experiment checkpointing and resuming
+- Environment interfaces for many popular benchmark suites
+- Supporting for running garage in diverse environments, including always up-to-date Docker containers
+
+## Why garage?
+
+garage aims to provide both researchers and developers:
+
+- a flexible and structured tool for developing algorithms to solve a variety of RL problems,
+- a standardized and [reproducible](ensure_your_experiments_are_reproducible.md) environment for experimenting and evaluating RL algorithms,
+- a collection of benchmarks and examples of RL algorithms.
```

Comment:
```suggestion
- A collection of benchmarks and examples of RL algorithms.
```

Reply:
It's a continued sentence so I use lowercase here.